authors/apps/articles/serializers.py

```diff
@@ -187,3 +187,17 @@ class CommentHistorySerializer(serializers.ModelSerializer):
     class Meta:
         model = Comments
         fields = ('id', 'body', 'created_at', 'updated_at')
+
+
+class EmailCheckSerializer(serializers.Serializer):
```

Comment:
This serializer already exists in authentications/serializers.py. I think you can just call the one in the authentications app if you need it in your views. That will help us keep our code DRY